packages/compiler-sfc/src/templateCompilerModules/srcset.ts

```diff
@@ -38,7 +38,7 @@ function transform(
         }
 
         const imageCandidates: ImageCandidate[] = value
-          .substr(1, value.length - 2)
+          .slice(1, value.length - 2)
```

Comment:
I think this becomes

```suggestion
          .slice(1, value.length - 1)
```